src/layouts/Login/LoginPage.tsx

```diff
@@ -219,16 +230,39 @@ export const LoginPage = (): JSX.Element => (
     <RestrictedGovtMasthead />
     <Grid {...LOGIN_GRID_LAYOUT}>
       <GridItem area="image" bgColor="primary.500">
-        <Flex h="100%" w="100%" alignItems="end" justifyContent="center">
-          <LoginImage maxH="100%" />
+        <Flex
+          h="calc(100vh - 5rem - 2rem)"
```

Comment:
why is this calculated this way? do note that this means some stuff won't display nicely and might overflow due to it having a fixed height now

Reply:
Svg position was really difficult! will rework once we have a centered svg instead